plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -67,6 +67,11 @@ const OWNER_REPO_SCHEMA = {
       title: 'Store path',
       description: 'GitHub store path in org/repo format',
     },
+    access: {
+      type: 'string' as const,
+      title: 'Access',
+      description: 'Who should have access, in org/team or user format',
```

Comment:
```suggestion
      description: 'Who should have access, in org/team or user format. To add multiple, use (,) to seperate',
```

Reply:
so you want support for multiple added here? I wasn't sure if required in the beginning.

I'm not really sure the value of adding multiple teams with admin permission.

Normally we would have an admins group and a group with write access, I didn't build this here as adding permission to the UI would make it more complicated in the initial PR

Ideally it would be something like this built but I didn't want to do it in this pull request:

![image](https://user-images.githubusercontent.com/21194782/93344450-9f689400-f829-11ea-95af-d8f27fad63df.png)

Reply:
Shit sorry, ignore me.

I've misread `const [, team] = access.split('/')` to think we were splitting on commas. 😐

It's been a long day. This looks good, Will merge.

Your recommendation for multiple is good and I think we will make something like this. But will chat tomorrow with the team for next steps.